conf/application.conf

```diff
@@ -130,7 +130,11 @@ microservice {
       bearerToken = test
       environment = test
     }
-
+    ecl-stubs {
```

Comment:
very minor comment but can we align config with microservice names, so instead of `ecl-stubs` call this `economic-crime-levy-stubs`

Reply:
Done.